sdk/computeschedule/arm-computeschedule/samples-dev/operationsOperationsListSample.ts

```diff
@@ -1,29 +0,0 @@
-// Copyright (c) Microsoft Corporation.
-// Licensed under the MIT License.
-
-import { ComputeScheduleClient } from "@azure/arm-computeschedule";
-import { DefaultAzureCredential } from "@azure/identity";
-
-/**
- * This sample demonstrates how to list the operations for the provider
- *
- * @summary list the operations for the provider
- * x-ms-original-file: 2024-10-01/Operations_List.json
- */
-async function operationsList(): Promise<void> {
-  const credential = new DefaultAzureCredential();
-  const subscriptionId = "00000000-0000-0000-0000-00000000000";
-  const client = new ComputeScheduleClient(credential, subscriptionId);
-  const resArray = new Array();
-  for await (const item of client.operations.list()) {
```

Comment:
no operation list?

Reply:
we skipped sample generation for this service as we have codegen issue here https://github.com/Azure/autorest.typescript/issues/3228